src/toil_scripts/gatk_germline/germline.py

```diff
@@ -1,688 +1,383 @@
                        help='Space delimited sample UUID and BAM file in the format: uuid url')
     parser_run.add_argument('--output-dir', default=None, help='Full path to directory or filename where '
-                                                               'final results will be output')    
-    parser_run.add_argument('-s', '--suffix', default='.bqsr', help='Additional suffix to add to the names of the output files')
+                                                               'final results will be output')
+    parser_run.add_argument('-s', '--suffix', default=None,
+                            help='Additional suffix to add to the names of the output files')
     Job.Runner.addToilOptions(parser_run)
     options = parser.parse_args()
 
     cwd = os.getcwd()
     if options.command == 'generate-config' or options.command == 'generate':
-        generate_file(os.path.join(cwd, 'gatk-preprocessing.config'), generate_config)
+        generate_file(os.path.join(cwd, 'config-toil-germline.yaml'), generate_config)
     if options.command == 'generate-manifest' or options.command == 'generate':
-        generate_file(os.path.join(cwd, 'gatk-preprocessing-manifest.tsv'), generate_manifest)
-
+        generate_file(os.path.join(cwd, 'manifest-toil-germline.tsv'), generate_manifest)
     # Pipeline execution
     elif options.command == 'run':
+        # Program checks
+        for program in ['curl', 'docker']:
+            require(next(which(program)), program + ' must be installed on every node.'.format(program))
+
         require(os.path.exists(options.config), '{} not found. Please run '
-                                             '"generate-config"'.format(options.config))
-        if not options.sample:
-            require(os.path.exists(options.manifest), '{} not found and no sample provided. Please '
-                                                       'run "generate-manifest"'.format(options.manifest))
-        # Parse config
-        parsed_config = {x.replace('-', '_'): y for x, y in yaml.load(open(options.config).read()).iteritems()}
-        inputs = argparse.Namespace(**parsed_config)
+                                                '"generate-config"'.format(options.config))
+        samples = []
         if options.manifest:
-            inputs.manifest = options.manifest
-    
-    inputs.cpu_count = multiprocessing.cpu_count() # FIXME: should not be called from toil-leader, see #186
-    inputs.memory = '15'
-    
-    Job.Runner.startToil(Job.wrapJobFn(batch_start, inputs, options.sample, options.output_dir, options.suffix), options)
+            samples.extend(parse_manifest(options.manifest))
+        if options.sample:
+            uuid, url, rg_line = options.sample.split()
+            samples.append((uuid, url, rg_line))
+
+        # Parse config
+        config = {x.replace('-', '_'): y for x, y in yaml.load(open(options.config).read()).iteritems()}
```

Comment:
I thought we were namespacing the configs? E.g., https://github.com/BD2KGenomics/toil-scripts/blob/master/src/toil_scripts/rnaseq_cgl/rnaseq_cgl_pipeline.py#L488, https://github.com/BD2KGenomics/toil-scripts/blob/master/src/toil_scripts/adam_uberscript/adam_uberscript.py#L585. We should change this to be consistent.